lldb/test/Shell/Commands/command-breakpoint-by-addr.test

```diff
@@ -0,0 +1,3 @@
+# RUN: %clang_host -g -O0 %S/Inputs/main.c -o %t.out
+# RUN: %lldb -b -o 'breakpoint set -a "main  +    26"' %t.out | FileCheck %s --check-prefix CHECK
```

Comment:
You should test a few versions in case the whitespace was ever made mandatory accidentally, on either side of the `+`.

Reply:
Done, will put them all in one lldb invocation